bokehjs/src/lib/models/dom/toggle_group.ts

```diff
@@ -0,0 +1,41 @@
+import {Action, ActionView} from "./action"
+import {RendererGroup} from "../renderers/renderer"
+import {ColumnarDataSource} from "../sources/columnar_data_source"
+import {Index as DataIndex} from "core/util/templating"
+import {enumerate} from "core/util/iterator"
+import * as p from "core/properties"
+
+export class ToggleGroupView extends ActionView {
+  override model: ToggleGroup
+
+  update(_source: ColumnarDataSource, i: DataIndex | null, _vars: object/*, formatters?: Formatters*/): void {
+    for (const [group, j] of enumerate(this.model.groups)) {
+      group.visible = i == j
```

Comment:
I know they are not required but I do think parens would improve readability here